packages/svelte/tests/signals/test.ts

```diff
@@ -1,10 +1,10 @@
 import { describe, assert, it } from 'vitest';
 import * as $ from '../../src/internal/client/runtime';
-import { derived } from '../../src/internal/client/reactivity/deriveds';
 import { effect, render_effect, user_effect } from '../../src/internal/client/reactivity/effects';
 import { source, set } from '../../src/internal/client/reactivity/sources';
 import type { Derived } from '../../src/internal/client/types';
 import { proxy } from '../../src/internal/client/proxy';
```

Comment:
Should this be proxies as the others directories are plural?

Reply:
very possibly, yeah. for now i'm just trying to extract all the reactivity changes out of `blockless` so that it becomes a bit more tractable